requirements_dev.txt

```diff
@@ -0,0 +1,2 @@
+-r requirments.txt
```

Comment:
Yeah, I though about that but didn't find the perfect solution. For now we can follow the approach with different virtualenvs to avoid blocking other progress. Meanwhile we can try good options for that.

I have added .travis.yml to run test on travis.